cromshell

```diff
@@ -449,9 +511,14 @@ function populateWorkflowIdAndServerUrl()
       grep ${WORKFLOW_ID} ${CROMWELL_SUBMISSIONS_FILE} > ${tmpFile}
       r=$?
       [[ ${r} -eq 0 ]] && WORKFLOW_SERVER_URL=$( awk '{print $2}' ${tmpFile} )
-
-      # If the user specified nothing, get the last workflow ID:  
+ else
+    if [[ -n "$userSpecifiedId" ]]; then
+			# The user specified somethign, but it's not a relative ID or a proper id.
+			# It must be an alias.
```

Comment:
accidental tabs

Reply:
Oh boy.  I forgot to de-tab the file.  Good catch.